zerver/lib/actions.py

```diff
@@ -2901,7 +2912,7 @@ def do_create_realm(string_id: Text, name: Text, restricted_to_domain: Optional[
                               "signups", realm.display_subdomain, signup_message)
     return realm
 
-def do_change_notification_settings(user_profile: UserProfile, name: str, value: bool,
+def do_change_notification_settings(user_profile: UserProfile, name: str, value: Union[bool, str],
```

Comment:
Same here.

Reply:
@roberthoenig This is needed because `notification_sound` is a string but the other `notification_settings` are all boolean values. Because of this, before, the type was assumed to be just `bool`, but if it was left like that mypy would fail since now a string `value` is possible.